divi/src/rpcwallet.cpp

```diff
@@ -34,6 +34,12 @@ using namespace json_spirit;
 
 int64_t nWalletUnlockTime;
 static CCriticalSection cs_nWalletUnlockTime;
+extern BlockMap mapBlockIndex;
+extern int64_t nReserveBalance;
+
+extern const std::string strMessageMagic = "DarkNet Signed Message:\n";
```

Comment:
Doesn't this definition make more sense inside obfuscation.cpp and then included in the other .cpp that require it via the extern declaration?

Reply:
done